pkgs/development/web/publii/default.nix

```diff
@@ -0,0 +1,109 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, makeShellWrapper
+, wrapGAppsHook
+, alsa-lib
+, at-spi2-atk
+, at-spi2-core
+, atk
+, cairo
+, cups
+, dbus
+, expat
+, glib
+, gtk3
+, libsecret
+, mesa
+, nss
+, pango
+, udev
+, xdg-utils
+, xorg
+}: let
+  intName = "Publii";
+  version = "0.42.0";
+
+in stdenv.mkDerivation {
+  pname = lib.strings.toLower intName;
+  inherit version;
```

Comment:
```suggestion
}:

stdenv.mkDerivation {
  pname = "publii";
  version = "0.42.0";
```

Reply:
What do you dislike on the solution without duplicating the name/hardcode it on multiple points?

Reply:
It is more complicated than it needs to be. The name is very unlikely to change this does not need to be variable.